usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -57,7 +58,7 @@ def construct_assistance_response(requested_award_dict: dict) -> OrderedDict:
             ("last_modified_date", get_date_from_datetime(transaction["_modified_at"])),
         ]
     )
-    transaction["_lei"] = award["_lei"]
+    transaction["_trx"] = award["_trx"]
```

Comment:
Does it make more sense to return the `transaction_id` from the transaction record itself rather than insert the ID from the award record? The end result is identical.

Reply:
Totally.  Was in autopilot and just mimicked what was there.